lib/src/auth.dart

```diff
@@ -131,15 +146,18 @@ abstract class Authenticator {
     return parsed;
   }
 
-  // Requests the authentication token from Reddit based on parameters provided
-  // in [accountInfo] and [_grant].
+  /// Requests the authentication token from Reddit based on parameters provided
+  /// in [accountInfo] and [_grant].
   Future _requestToken(Map<String, String> accountInfo) async {
     // Retrieve the client ID and secret.
     String clientId = _grant.identifier;
     String clientSecret = _grant.secret;
+    String userInfo = null;
+
+    if ((clientId != null) && (clientSecret != null)) {
```

Comment:
nit: Unnecessary parentheses. While generally useful if precedence rules are unclear, I'd argue this condition is simple enough the parentheses don't improve clarity.